setup.py

```diff
@@ -278,7 +310,8 @@ def parse_long_description(filename, tag=False):
 
     def reduce_header_level():
         # PyPI doesn't like title to be underlined with =
-        readme_lines[1] = readme_lines[1].replace("=", "-")
+        if tag:
+            readme_lines[1] = readme_lines[1].replace("=", "-")
```

Comment:
Its in the right place, not dropping level is specific to tags and this commit is adding branch level changing.